charts/datahub/subcharts/datahub-mce-consumer/templates/deployment.yaml

```diff
@@ -67,14 +65,14 @@ spec:
           livenessProbe:
             httpGet:
               path: /actuator/health
-              port: 9090
+              port: actuator
```

Comment:
Can you please also review Mae-consumer deployment.yaml?
The liveness and readiness probes are using `http` port

Reply:
the same thing is exposed on 9091 on MAE, and called `http`, I called it actuator, because http is less descirptive.

Reply:
Can we make it consistent then? Call all of them `actuator`

Reply:
renamed this one to http